docs/tutorials/getting_started/interactions.rst

```diff
@@ -0,0 +1,34 @@
+Accessing the interactions
+==========================
+
+This tutorial will show you briefly how to access the detailed interaction
+results corresponding to the tournament.
+
+As shown in :ref:`getting-started` let us create a tournament except that,
+this time, we will set the with_interactions parameter. (We will also reduce
+the number of turns per match to 3 and we'll run only a single repetition)::
+
+    >>> import axelrod as axl
+    >>> strategies = [
+    ...     axl.Cooperator(), axl.Defector(),
+    ...     axl.TitForTat(), axl.Grudger()]
+    >>> tournament = axl.Tournament(
+    ...     strategies, turns=3, repetitions=1, with_interactions=True)
+    >>> results = tournament.play()
+
+We can now view the detailed interactions that occurred between the players
+for each turn in each match. (The interactions are stored within a dictionary,
+so we'll add a sort to show the entries in a sensible order).::
+
+    >>> for key in sorted(tournament.interactions):
+    ...     print('%s: %s' % (key, tournament.interactions[key]))
+    (0, 0): [('C', 'C'), ('C', 'C'), ('C', 'C')]
```

Comment:
Sounds great. I think having the match objects returned is important
because at some point down the line or indeed for potential other
tournament types those match objects could have other information.

I'll look at it all closely next week (on Monday I've got some train time
:+1:).

On Thu, 3 Mar 2016, 19:31 Owen Campbell, <notifications@github.com> wrote:

> In docs/tutorials/getting_started/interactions.rst
> <https://github.com/Axelrod-Python/Axelrod/pull/474#discussion_r54933895>:
>
> > +
> > +    >>> import axelrod as axl
> > +    >>> strategies = [
> > +    ...     axl.Cooperator(), axl.Defector(),
> > +    ...     axl.TitForTat(), axl.Grudger()]
> > +    >>> tournament = axl.Tournament(
> > +    ...     strategies, turns=3, repetitions=1, with_interactions=True)
> > +    >>> results = tournament.play()
> > +
> > +We can now view the detailed interactions that occurred between the players
> > +for each turn in each match. (The interactions are stored within a dictionary,
> > +so we'll add a sort to show the entries in a sensible order).::
> > +
> > +    >>> for key in sorted(tournament.interactions):
> > +    ...     print('%s: %s' % (key, tournament.interactions[key]))
> > +    (0, 0): [('C', 'C'), ('C', 'C'), ('C', 'C')]
>
> I'll have a go at using the match objects and some kind of function
> (possibly a property) to return a pretty version of the interactions from
> those objects.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/474/files#r54933895>.
>